packages/react-scripts/config/config.js

```diff
@@ -0,0 +1,81 @@
+// @remove-on-eject-begin
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+// @remove-on-eject-end
+'use strict';
+
+const fs = require('fs');
+const chalk = require('chalk');
+const paths = require('./paths');
+
+/**
+ * Get the baseUrl of a compilerOptions object.
+ *
+ * @param {Object} options
+ */
+function getBaseUrl(options = {}) {
+  const baseUrl = options.baseUrl;
+
+  if (!baseUrl) {
+    return null;
+  }
+
+  if (baseUrl !== 'src' && baseUrl !== './src') {
+    console.error(
+      chalk.red.bold(
+        "You tried to set baseUrl to anything other than 'src'. This is not supported in create-react-app and will be ignored."
+      )
+    );
+
+    return null;
+  }
+
+  return 'src';
+}
+
+/**
+ * Get the paths of a compilerOptions object.
+ *
+ * @param {Object} options
+ */
+function getPaths(options = {}) {
```

Comment:
Do we need that? I cannot seem to see the `paths` value being used at all. Also you're not returning `paths` if it's defined, the function currently either returns an empty array of undefined.

Reply:
This is still work in progress to support paths/aliases but I wanted to get the test working first.